tests/command.exitOverride.test.js

```diff
@@ -175,19 +175,18 @@ describe('.exitOverride and error details', () => {
     expectCommanderError(caughtErr, 0, 'commander.version', myVersion);
   });
 
-  // Have not worked out a cleaner way to do this, so suppress warning.
-  // eslint-disable-next-line jest/no-test-callback
-  test('when executableSubcommand succeeds then call exitOverride', (done) => {
+  test('when executableSubcommand succeeds then call exitOverride', async() => {
     const pm = path.join(__dirname, 'fixtures/pm');
```

Comment:
To be explicit about having assertions (as well as calling resolve eventually), I think add a first line:

```
expect.hasAssertions();
```

(This could have been done in the previous version of the test too! 😄  )

Reply:
Good idea, i've added it.